src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -471,227 +471,225 @@ <h1>{{ _("Export transactions to CSV") }}</h1>
             }
         }
 
-    }
-
-    /**
-     * Fetches txlist from the Specter API and loads the result into TxRowElement
-     * @param self - The TxTableElement itself.
-    */
-    async function fetchTxItems(self) {
-        self.callId++;
-        // Clean up from existing transaction list
-        self.summaryRow.classList.add('hidden');
-        self.emptyRow.children[1].innerText = '{{ _("Fetching transactions...") }}';
-        self.emptyRow.classList.remove('hidden');
-        self.tbody.querySelectorAll('tx-row').forEach((row) => {row.remove()});
-        if (!self.listType) {
-            self.emptyRow.children[1].innerText = '{{ _("Failed to fetch transactions...") }}';
-            return
-        }
-
-        if (self.listType == "txlist") {
-            self.txlistViewBtn.classList.add("checked");
-        } else {
-            self.utxoViewBtn.classList.add("checked");
-        }
-
-        // Select the correct URL based on the list type and selected wallet
-        let url;
-        switch (self.listType) {
-            case "txlist":
-                if (self.wallet) {
-                    self.export.href = `{{ url_for('wallets_endpoint.tx_history_csv', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", self.wallet);
-                    url = `{{ url_for('wallets_endpoint.txlist', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", self.wallet);
-                } else {
-                    self.export.href = `{{ url_for('wallets_endpoint.wallet_overview_txs_csv') }}`;
-                    url = `{{ url_for('wallets_endpoint.wallets_overview_txlist') }}`;
-                }
-                break;
-            case "utxo":
-                if (self.wallet) {
-                    self.export.href = `{{ url_for('wallets_endpoint.utxo_csv', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", self.wallet);
-                    url = `{{ url_for('wallets_endpoint.utxo_list', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", self.wallet);
-                } else {
-                    self.export.href = `{{ url_for('wallets_endpoint.wallet_overview_utxo_csv') }}`;
-                    url = `{{ url_for('wallets_endpoint.wallets_overview_utxo_list') }}`;
-                }
-                break
-            default:
-                self.export.href = ``
-                self.emptyRow.children[1].innerText = '{{ _("Failed to fetch transactions...") }} ';
+        /**
+         * Fetches txlist from the Specter API and loads the result into TxRowElement
+        */
+        async fetchTxItems() {
+            self.callId++;
```

Comment:
still self? and everywhere below as well.

Reply:
Yes sorry I was in a hurry because I had to go and thought I could finish before leaving but clearly was a stupid idea